itests/common.bash

```diff
@@ -252,6 +252,12 @@ _profile() {
   fi
 }
 
+
+_wait_for_poller_start() {
```

Comment:
Checkout `_wait`. You can do a similar thing with a startup message so it's a bit more flexible and not time dependent.

Reply:
i like it, less chances of a race condition.